src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -218,39 +227,121 @@ class ActionDelegate {
     }
 
     /**
-     * Changes behavior of a given field on given transition if certain condition is being met.
+     * Changes behavior of a given field on given transition if certain condition is being met or on each transition containing a field.
      * <br>
-     * Example:
+     * Example 1:
      * <pre>
      *     condition: f.conditionId,
      *     text: f.textId,
      *     transition: t.transitionId;
      *
      *     make text,visible on transition when { condition.value == true }* </pre>
-     * This code will change the field <i>text</i> behaviour to <i>visible</i> when fields <i>condition</i> value is equal to <i>true</i>
+     * This code will change the field <i>text</i> behaviour to <i>visible</i> when fields <i>condition</i> value is equal to <i>true</i>.
+     *
+     * Example 2:
+     * <pre>
+     *      text: f.textId;
+     *
+     *      if (condition)
+     *          make text, visible on each
```

Comment:
what is the syntax? the doc says `each` the variable used in the code is called `each`, but the string value returned is `each transition` if I am not mistaken. Is this intended?

Reply:
each represents "each transition", we just created it as a dummy Closure returning random string so the syntax wouldn't be complex or hard to remember (otherwise syntax would be `make text, visible on each()`).
`make text, visible on each` is simplified `make text, visible on each transition`. The returned string just doesn't do anything.